client/src/components/App/index.js

```diff
@@ -2,24 +2,29 @@ import React from 'react'
 import './styles.css'
 import Header from 'src/components/Header'
 import CreateCardFormContainer from 'src/containers/CreateCardFormContainer'
-import TodoCardListContainer from 'src/containers/TodoCardListContainer'
+import TodoCardList from 'src/components/TodoCardList'
+import TodoCard from 'src/components/TodoCard'
```

Comment:
@DDKatch when we are talking about List and items, we usually use such pattern:

```
TodoCardList/
  Item/
    index.js
  index.js
```

So, dependent Item is under List hierarchy because Item isn't used separately 😉